Source/Fuse.Models/Tests/Model.Test.uno

```diff
@@ -11,6 +11,53 @@ namespace Fuse.Models.Test
 {
 	public class ModelTest : ModelTestBase
 	{
+
+		[Test]
+		public void Args_Each()
```

Comment:
Unusual naming convention, we usually don't have the underscore.

Reply:
👍